README.md

```diff
@@ -323,3 +324,15 @@ In decending order of priority (top ones are more important):
       Maybe we can find a way, to wait for specific events to be triggered on the page. For example: Wait for all animations on the page to complete before taking the screenshot.
 
 If you find a bug or think some feature is missing, don't hesitate to submit an issue.
+
+## Building the Project
```

Comment:
I think renaming the section to "Contributing" or something similar would be better.
The current headline could lead to confusion otherwise.

Reply:
👍 56f1dff